balancesExporter/trie/trieWrapper.go

```diff
@@ -0,0 +1,54 @@
+package trie
+
+import (
+	"github.com/ElrondNetwork/elrond-go/common"
+	"github.com/ElrondNetwork/elrond-go/state"
+)
+
+type trieWrapper struct {
+	trie common.Trie
+}
+
+func newTrieWrapper(t common.Trie) *trieWrapper {
+	return &trieWrapper{trie: t}
+}
+
+func (tw *trieWrapper) IsRootHashAvailable(rootHash []byte) bool {
+	_, err := tw.trie.GetAllLeavesOnChannel(rootHash)
```

Comment:
misleading name. This actually tries to get all trie nodes on a channel. It is not safe to use this function in this way due to resources it might free only at a latter time.
Suggestion to just to a plain get here for the roothash key from the storer.

Reply:
Fixed :pray: